src/plugins/favEmojiFirst.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Vencord, a modification for Discord's desktop app
+ * Copyright (c) 2023 Vendicated and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import { Emoji } from "@api/MessageEvents";
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+import { findByPropsLazy } from "@webpack";
+
+const DisambiguatedEmojiContext = findByPropsLazy("getDisambiguatedEmojiContext");
```

Comment:
```suggestion
const EmojiStore = findStoreLazy("EmojiStore");
```

Also this is now used by 3 plugins (EmoteCloner, FakeNitro, Yours, maybe more?), so perhaps it could be moved to webpack commons and properly typed if you would like doing that